Apps/WebClient/src/ClientApp/app/models/timelineEntry.ts

```diff
@@ -1,4 +1,5 @@
 import MedicationStatement from "@/models/medicationStatement";
```

Comment:
I am changing the whole way we load timeline entries. So this file might complelty go away.